### PR TITLE
fix: moved separator line from footer jet to app badges jet

### PR DESCRIPTION
**Pull request by PeculiarGoat**

### CODE REVIEW

File: site/templates/footer/app-badges.jet [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {{block appBadges()}}
   {{app_link_ios := config("app_link_ios")}}
   {{app_link_android := config("app_link_android")}}
-  {{if config("app_link_ios") != "" || config("app_link_android") != ""}}
+  {{if config("app_link_ios") == "" || config("app_link_android") == ""}}
```
Comment (stajs) [inline site/templates/footer/app-badges.jet:6]: Is this right? If the app links are blank they shouldn't be shown should it?

**Review by stajs — COMMENTED** [review]
Just checking...

File: site/templates/footer/app-badges.jet (hunk 1 above)
Comment (PeculiarGoat) [reply site/templates/footer/app-badges.jet:6]: Oh snap!  Good catch... no, I was fiddling that for testing.

**Comment by PeculiarGoat** [comment]
@stajs ready!

**Review by stajs — APPROVED** [review]
:shipit:

**Pull request merged**